tests/sls/common/timezone.py

```diff
@@ -0,0 +1,18 @@
+import subprocess
+
+from tests.util import Failure, Success
+
+
+def run():
+    ret = subprocess.run(
+        ['date'],
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE
+    )
+
+    stdout = ret.stdout.decode('utf-8')
+
+    if ret.returncode == 0 and 'UTC' in stdout:
+        return Success('Date is in UTC')
```

Comment:
I was reading an article about testing yesterday which reminded me of expected-failure tests, which is IMO a nicer way to handle the macOS situtation. Roughly, instead of just returning `Success` here, use `sys.platform` to check if we're on macOS, and if so, return `Failure` in that case with an appropriate message; and vice-versa for the `Failure`. This inverts the expectations and lets us run the `sls.common` test unconditionally.

Note that in the .travis.yml file, I believe `python` isn't supported on macOS, so don't update those entries to add the python3.5 spec. However, we may need to install Python 3.5 manually with `brew` on the OS X builders; make a separate commit and see if it's necessary before adding that.


It may be a little tricky to add so I'll make it optional, for bonus points :)